src/components/ProductShow/index.js

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 
 function ProductShow() {
-  return <div>ProductShow</div>;
+  return <div>ProductShow
+
+<RelatedItems />
+  </div>;
```

Comment:
think the return statement needs to have parentheses around the divs if it's multi-line. Also, the indentation of line 6